ui/i18n/messages.json

```diff
@@ -171,6 +171,29 @@
   "groupManagement.removeFrom": "Remove from '{0}' group",
   "groupManagement.removeUserFromGroup.confirm": "Do you really want to remove {0} from this group?",
   "groupManagement.users.heading": "Users",
+  "htmlEditor.align.center": "Align center",
```

Comment:
We are adding new labels. Please add the `translation` tag to the ticket and, after merging, create a ticket for Manon to trigger the auto translations. If could even to this and then backport the translation labels in the PR for 3.0.x. It would save us some time.